packages/modules/manage-versions/src/components/ManageVersions/ChangesTab/ChangesTab.tsx

```diff
@@ -62,6 +60,15 @@ const ChangesTab = (props: ChangesTabProps) => {
             Header: stringsOverrides.description,
             accessor: "description",
           },
+          {
+            id: "CHANGED_FILES",
+            Header: stringsOverrides.changedFiles,
+            Cell: (props: CellProps<Changeset>) => {
+              const changedFiles =
+                props.row.original.synchronizationInfo.changedFiles;
```

Comment:
`synchronizationInfo` can be `null` 
![image](https://user-images.githubusercontent.com/36186912/220883089-be802a54-f8a6-41ca-9456-0f1dfce8e580.png)
![image](https://user-images.githubusercontent.com/36186912/220883015-cf40062e-3a3a-4c64-8692-19fbcf5cc71e.png)
![image](https://user-images.githubusercontent.com/36186912/220883089-be802a54-f8a6-41ca-9456-0f1dfce8e580.png)
Change to `synchronizationInfo?.changedFiles`

Reply:
done